src/transformers/modeling_camembert.py

```diff
@@ -51,12 +51,6 @@
             model. Initializing with a config file does not load the weights associated with the model, only the
             configuration.
             Check out the :meth:`~transformers.PreTrainedModel.from_pretrained` method to load the model weights.
-        output_attentions (:obj:`bool`, `optional`, defaults to :obj:`None`):
```

Comment:
This is the documentation of the init, not the forward, so this shouldn't have been added here.